tripy/tripy/frontend/utils.py

```diff
@@ -35,7 +35,7 @@ def tensor_from_shape_like(arg: ShapeLike) -> "tripy.Tensor":
     from tripy.frontend.trace.ops.reshape import Reshape
 
     if not arg:
-        return Tensor([], dtype=int32)
+        return Tensor.create_directly([], dtype=int32)
```

Comment:
Are these just for performance?

Reply:
Yeah, I figured most internal uses of the constructor shouldn't need to go through the registry.

Reply:
Yup, makes sense.